tests/integration/targets/ios_lag_interfaces/vars/main.yaml

```diff
@@ -1,16 +1,11 @@
 ---
 merged:
-  before:
-    - name: GigabitEthernet0/0
```

Comment:
For all the interface modules, we decided to populate all the interfaces present in the device ( though they do not have any configs under them) in the after and before keys. Can you please check this point with the team?

Reply:
Yes, that is correct but specific to this module the facts would be misleading if we look at it that way,
The facts for `lag_interfaces` is a bit twisted i.e we have options Port-channels as the parent attribute under which all the other interfaces are termed as members, showing all the other interfaces in the top level with no data in members is basically wrong in terms of facts specific to lag_interfaces. This makes me think only blank Port-channels should be visible.

Reply:
@GomathiselviS so now
for a configuration as 
```
interface Port-channel11
interface Port-channel22
interface Port-channel33
interface GigabitEthernet0/1
 shutdown
 channel-group 11 mode active
interface GigabitEthernet0/2
 shutdown
 channel-group 22 mode active
interface GigabitEthernet0/3
 shutdown
 channel-group 22 mode passive
interface GigabitEthernet0/4
 shutdown
 channel-group 22 mode active link 20
interface GigabitEthernet0/5
 shutdown
interface Port-channel44
```
We have 4 Port-channels and 5 GigEth and GigEth0/5 is not under a channel group 
we would produce the facts like -
```
"parsed": [
        {
            "members": [
                {
                    "member": "GigabitEthernet0/1",
                    "mode": "active"
                }
            ],
            "name": "Port-channel11"
        },
        {
            "members": [
                {
                    "member": "GigabitEthernet0/2",
                    "mode": "active"
                },
                {
                    "member": "GigabitEthernet0/3",
                    "mode": "passive"
                },
                {
                    "link": 20,
                    "member": "GigabitEthernet0/4",
                    "mode": "active"
                }
            ],
            "name": "Port-channel22"
        },
        {
            "members": [
                {}
            ],
            "name": "Port-channel33"
        },
        {
            "members": [
                {}
            ],
            "name": "Port-channel44"
        }
    ]
```
Not making a group for GigEth0/5 as it's not a valid group as per lag_interfaes and has no option to have _members_ within it.